discovery_engine/lib/src/domain/event_handler.dart

```diff
@@ -308,6 +313,7 @@ class EventHandler {
       _openDbBox<ActiveDocumentData>(activeDocumentDataBox),
       _openDbBox<Uint8List>(engineStateBox),
       _openDbBox<ActiveSearch>(searchBox),
+      _openDbBox<Set<String>>(sourcesBox),
```

Comment:
After merging #284 this will change to `Set<Sources>` same in other places.

Reply:
thanks, will adjust once that's merged.